docs/docs/running-hello-world.md

```diff
@@ -151,9 +151,13 @@ In the previous section, you run the sample application in [mock mode](enclave-m
 suggests, isn't secure at all.
 
 This section describes how to build and run the sample in other [enclave modes](enclave-modes.md), namely
-simulation, debug, and release. Note that only release mode provides confidential guarantees of SGX.
+simulation, debug, and release.
 
-*First, ensure that your system fulfills the [system requirements](enclave-modes.md#system-requirements).*
+Only the release mode provides confidential guarantees of SGX. You can learn to develop enclaves on Windows, macOS, 
```

Comment:
The release mode statement doesn't belong with the rest of the paragrah. In fact, perhaps it should be its our info box.

Reply:
Instead, I have removed the sentence. It's better to avoid info boxes on this first tutorial to avoid affecting the flow. We already mention on the enclave-modes page that only the release mode is confidential. We also link to it from this tutorial. I believe that is sufficient.